lib/membrane_rtc_engine/endpoints/webrtc_endpoint.ex

```diff
@@ -217,7 +217,11 @@ defmodule Membrane.RTC.Engine.Endpoint.WebRTC do
   end
 
   @impl true
-  def handle_notification({:removed_tracks, tracks}, _from, _ctx, state) do
+  def handle_notification({:removed_tracks, _tracks}, _from, _ctx, %{recv_only: true} = _state),
+    do: raise("Peer is not allowed to send any media")
```

Comment:
This should never happen and it tests or ensures that webrtc plugin works properly. I would delete this